docs/getting_started.md

```diff
@@ -1,58 +1,122 @@
 # Getting started with `build_runner`
 
-`build_runner` can be used as a development server.
+To use `build_runner`, you need a 2.x (dev channel or fresher) version of
+the Dart SDK.
 
-1. Add a `dev_dependency` on `build_runner: ^0.7.0`
-2. Add a `dev_dependency` on `build_web_compilers: ^0.2.0`
-3. Run `pub get`
-4. To start the server run `pub run build_runner serve`
+* [Automated installers](https://www.dartlang.org/install#automated-installation-and-updates)
+* [Direct downloads](https://www.dartlang.org/install/archive#dev-channel)
 
-The first build will be the slowest. After that assets are cached on disk and
-incremental builds will be faster. While the `serve` command is running any
-changes you save to files will trigger rebuilds automatically.
+If you have issues using `build_runner`, see the
+[Troubleshooting section](#troubleshooting), below.
 
-## Available commands
 
-In addition to `serve` you can use:
+## Using `build_runner` as a development server
 
-- `build` - run a single build and exit. This is most useful if your build also
-  generates outputs to your source directory. With `--output <dirname>` this
+1. Edit your package's **pubspec.yaml** file,
+   removing all transformers and
```

Comment:
removing transformers is optional - people can try it out without breaking their existing `pub serve` workflow.